server_tests/test/clientTests.js

```diff
@@ -32,7 +31,8 @@ describe("RUST API TEST SUITE", function() {
     var isBublcam = testModel === camModels.BUBLCAM1_0 || testModel === camModels.BUBLCAM1_2 || testModel === camModels.BUBLMOCK;
     var isMock = testModel === camModels.BUBLMOCK;
     var testViaWifi = process.env.SCARLET_TEST_WIFI === '1';
-    function shouldNotGetHere(err) {
+    var timeoutValue = testViaWifi? 45000 : 30000;
```

Comment:
USB is significantly faster, yes, but only by tens of milliseconds; the purpose of this timeout is to account for capture delays. It would be better for each individual test to select its own timeout (most would be a couple seconds, captures would depend on the number of photos taken), so this is a bit of a weird catch-all... but I guess the point is, for any of the tests that use `timeoutValue`, is there a conceivable reason why they could take longer than 30 seconds?